active/NavigationView/BottomItems/BottomMenuItems.md

```diff
@@ -0,0 +1,143 @@
+
+# Background
+Today, [NavigationView](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/navigationview) provides a place in its [Pane](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/navigationview) to show any number of items.
+The primary purpose of these items is to be a list of navigation links within the app that hosts NavigationView, likely as a root element.
+This navigation list is top aligned when NavigationView is shown in one of its left [display modes](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/navigationview#display-modes), and left aligned when NavigationView is in its top [display mode](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/navigationview#display-modes).
+NavigationView provides a built-in [Settings item](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.navigationview.settingsitem?view=winui-2.3), which is always positioned at the bottom or right of the Pane, and a [PaneFooter](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.navigationview.panefooter?view=winui-2.3) area that positions content just before (either above or to the left) of the Settings item.
+Although an app developer can place any content into PaneFooter, including other NavigationViewItems, this content will not participate in the selection model and animations of the primary navigation list and Settings item.
+
+_Open question:_ Should PaneFooter be deprecated because this new API is a direct replacement?
+
+Some apps may choose to visually group navigation items into top-aligned and bottom-aligned lists.
+While this grouping is possible today using navigation view's [PaneFooter](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.navigationview.panefooter?view=winui-2.3) property, the approach has interaction downsides that this feature will correct.
+
+![NavigationView in Left display mode showing FooterMenuItems](FooterItems_Left.png)
+
+![NavigationView in Top display mode showing FooterMenuItems](FooterItems_Top.png)
+
+By adding FooterMenuItems, we'll unify the [Settings item](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.navigationview.settingsitem?view=winui-2.3) with other navigation items in its immediate proximity.
```

Comment:
Yeah, I think it would be useful to expand on this. Unifying with the settings item means that the menu items that you specify to be in the footermenu (bottom menu items) will be formed into their own list with the settings item, apart from the list of top menu items. 

As you can see in the keyboarding section, the narrator will read bottom menu items (including the settings item) as members of its own list, separately indexed from the top menu items. In addition to that, they will be unified visually with the rest of the navigationview - bottom items will now participate in the same selection model and animations as the rest of the menu items (top & bottom). 

Does that make sense? I'll work on clarifying this section a bit.